docs/attribution.md

```diff
@@ -1,19 +1,18 @@
 # Attribution
 
-
-Attribution is required for many of our data providers. Example language is provided below, but you are responsible for researching each project to follow their license terms. More details are available on the [Data Sources](data-sources.md) page.
+Attribution is required for using Mapzen's hosted vector tile service as well as for many of our data providers. Example language is provided below, but you are responsible for researching each project to follow their license terms. More details are available on the [Data Sources](data-sources.md) and the [Mapzen rights](https://mapzen.com/rights) pages.
 
 **Required attribution:**
 
->  © OpenStreetMap contributors, Who's On First
+>  © Mapzen, OpenStreetMap contributors, Who's On First
```

Comment:
And same comment from the terrain docs review where this is now for combination of the hosted service (e.g. Mapzen credit) and the data (existing credit).